### ISSUE TITLE
Add DNS SDE Extension Option

### ISSUE THREAD

**Issue by Greigh** [AdguardTeam/AdGuardHome#7467]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### The problem

It gets old just having a website just show an error with having to go into the AdGuard settings and it would be nice for a Home product to allow the family guests the option to request an allowlist without having to ask what page they need unblocked or just what's going on, on their device. I have had people complain or just simply not want to use my network because it won't allow them through to a certain website that might not need to be blocked. 

### Proposed solution

It would be nice to have it redirect to a full fledged webpage where a user on the network could request the website to be unblocked, etc. It would explain to guests exactly what's going on, etc. You could use the DNS SDE and have it redirect to a webpage created with HTML/CSS/JS with the option in settings to allow requests. Additionally, a link to add to allowlist. It could be super simple to setup since the SDE already contains most of the code.

**Comment by saint-lascivious** [comment]
Variations of this suggestion frequently pop up on the trackers of domain filters. It seems like a great idea until reality slaps you in the face with how things actually work.

Unless you want to effectively MITM yourself, anything even remotely current is going to feel some ways about an attempt to redirect a secure session, and just flat out won't let that happen.

The resulting inability to arbitrarily redirect a secure session is by and large a major reason why HTTPS exists.
